### ISSUE TITLE
Feature request: Support MAF

### ISSUE THREAD

**Issue by Marxsal** [tagspaces/tagspaces#578]
It's been suggested that MHT could be used to capture web page content similar to Ever-Note. But MHT content is huge, even when you save only a sub-selection. MAF creates much smaller files (33% or 16% after tweaking). If a full support isn't possible, then even having a thumbnail of the MAF contents would be useful.

**Comment by Zireael07** [comment]
MHT is still smaller than HTML :P But yeah, good idea.

**Comment by Marxsal** [comment]
In my test,  MHT was only 15% less than the same article captured in scrapbook. 

So the original captured in scrapbook was about 1.85 megs. The original in MHT was 1.58 megs. If I took the scrapbook version and then applied MHT the result was only about a meg.

But if I saved the original directly as MAFF it took only 473kb -- about a 3rd of MHT thanks to compression.

Of course, all of these are still too big for a simple article. When I made my own version, clipping just the html and images that I needed, the resulting article was only 95kb. If I took that article and applied MHT, the result was actually a little larger (105kb). It would be great if there was a clipper that would ignore all lthe embedded fonts and unused images.
